.changeset/dull-clouds-sparkle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@pantheon-systems/gatsby-wordpress-starter': major
```

Comment:
Since this does not appear to be a breaking change, I think this should be a minor change rather than a major change.